compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -87,51 +98,44 @@ createProjectPackageDb projectRoot (disableScenarioService -> opts) modulePrefix
       -- data-dependency but that seems acceptable.
       -- See https://github.com/digital-asset/daml/issues/4218 for more details.
       -- TODO Enforce this with useful error messages
-      registerDepsInPkgDb depsDir dbPath
+      -- registerDepsInPkgDb depsDir dbPath
```

Comment:
As far as I could tell those never make it to the `.daml/dependencies` dir because of [DA.Cli.Damlc.DependencyDb#126](https://github.com/digital-asset/daml/blob/data-deps-deps/compiler/damlc/lib/DA/Cli/Damlc/DependencyDb.hs#L126). I _did_ need to recache the package db right at the start for `daml-stlib` and `daml-prim` to be picked up, see [here](https://github.com/digital-asset/daml/pull/13205/files#r823890157)

Reply:
hm I checked again and the recache at the top isn't needed, and now I can't remember why I added it 🤷